### PR TITLE
Refactor Colwise

### PR DESCRIPTION
**Pull request by eliascarv**

### CODE REVIEW

File: src/transforms.jl [hunk 1]
```diff
@@ -273,10 +283,10 @@ function applyfeat(transform::Colwise, feat, prep)
   newfeat = 𝒯 |> Tables.materializer(feat)
 
   # cache values for each column
-  fcache = last.(vals)
```
Comment (juliohm) [inline src/transforms.jl:276]: Isn't fcache the correct term in this context? We are inside the applyfeat, right?

**Review by juliohm — COMMENTED** [review]
Amazing work @eliascarv , just left a small comment regarding the rename of a fcache variable. Can you please double check?

File: src/transforms.jl (hunk 1 above)
Comment (eliascarv) [reply src/transforms.jl:276]: In this case, fcache is the tuple of colcaches and snames:
```julia
fcache = (caches, snames)
```

**Comment by juliohm** [comment]
Perfect. Thanks for double checking. Feel free to merge it. :)

Em sex., 23 de set. de 2022 08:07, Elias Carvalho ***@***.***>
escreveu:

**Pull request merged**